### ISSUE TITLE
Error when executing roslauch concert_gazebo concert.launch

### ISSUE THREAD

**Issue by manuelvogel12** [ADVRHumanoids/concert_description#29]
Using the newest version of concert_description and modular I get the following error when executing `roslauch concert_gazebo concert.launch`
```
[INFO] [URDF_writer]:  Module added to URDF: end_effector_E (end_effector)
[INFO] [URDF_writer]:  Module added to URDF: end_effector_E (end_effector)
[INFO] [core]:  signal_shutdown [atexit]
[INFO] [core]:  signal_shutdown [atexit]
Traceback (most recent call last):
  File "/home/user/tum_integration_ws/ros_src/concert_description/concert_examples/concert_example.py", line 81, in <module>
    write_file_to_stdout(urdf_writer, homing_joint_map)
  File "/home/user/tum_integration_ws/src/modular/src/modular/URDF_writer.py", line 3941, in write_file_to_stdout
    content = urdf_writer.write_srdf(homing_map)
  File "/home/user/tum_integration_ws/src/modular/src/modular/URDF_writer.py", line 3721, in write_srdf
    srdf_string = self.control_plugin.write_srdf(builder_joint_map)
  File "/home/user/tum_integration_ws/src/modular/src/modular/URDF_writer.py", line 222, in write_srdf
    end_effectors += filter(lambda item: item is not None, [self.add_gripper_to_srdf(et_root=None,
  File "/home/user/tum_integration_ws/src/modular/src/modular/URDF_writer.py", line 694, in add_gripper_to_srdf
    if len(module.finger_names) == 0:
AttributeError: 'ModuleNode' object has no attribute 'finger_names'
[INFO] [core]:  signal_shutdown [atexit]
Could not load launch file: /home/user/tum_integration_ws/ros_src/concert_description/concert_gazebo/launch/modular.launch:71: <param> command failed (exit status 1)
```

Also, is `advrhumanoids/iit-dagana-ros-pkg` now another requirement?

**Comment by EdoardoRomiti** [comment]
Hi @manuelvogel12 , sorry this bug comes from a typo in the previous commits of `modular_hhcm`. I just fixed it in ADVRHumanoids/modular_hhcm@977b085 on the master branch.

> Also, is advrhumanoids/iit-dagana-ros-pkg now another requirement?

No it's not, it's just an external resource you might want to use or not. Is needed only if you use the `dagana` end-effector module when building your URDF. 
If you don't and don't want to use it just check that the modular config file: [ADVRHumanoids/modular_hhcm/src/modular/config_file.yaml](https://github.com/ADVRHumanoids/modular_hhcm/blob/master/src/modular/config_file.yaml) is configured correctly. In case you don't want to use the dagana just remove the `dagana_path` key from the config and you shouldn't see any errors

**Comment by manuelvogel12** [comment]
Thank you, it works now again
